src/Files.App.Controls/Sidebar/SidebarView.xaml

```diff
@@ -118,7 +117,6 @@
 			Visibility="{Binding ElementName=PaneRoot, Path=Visibility, Mode=OneWay}">
 			<UserControl
 				x:Name="SidebarResizerControl"
-				AutomationProperties.Name="{helpers:ResourceString Name=ResizeElementControl/AutomationProperties/Name}"
```

Comment:
Is this resource still being used, or can the string be removed?

Reply:
I cant even find the resource string